src/hooks/screen/useCreateChat.ts

```diff
@@ -93,17 +124,20 @@ const useCreateChat = () => {
     }
   };
 
-  const handleAnonymousMessage = async (selectedUser) => {
+  const handleAnonymousMessage = async (selectedUser, channelId = null, context = null) => {
     if (!selectedUser?.allow_anon_dm) {
       SimpleToast.show('This user does not allow anonymous messages');
       return;
     }
 
     try {
       const response = await getOrCreateAnonymousChannel(
-        selectedUser?.user_id || selectedUser.userId
+        selectedUser?.user_id || selectedUser.userId,
+        channelId,
+        context
       );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `handleAnonymousMessage` has been updated to accept additional parameters `channelId` and `context`. These changes seem fine, but there's no check for the validity of these new parameters. It would be a good idea to add checks or default values for these parameters to prevent potential issues.

```diff
- const handleAnonymousMessage = async (selectedUser, channelId = null, context = null) => {
+ const handleAnonymousMessage = async (selectedUser, channelId = '', context = {}) => {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->